### ISSUE TITLE
Plugin Setting

### ISSUE THREAD

**Issue by ghost** [jrrdnx/craft-helpdesk-support#3]
@jrrdnx I have just found another bug of new version :)
`Class jrrdnx\helpdesksupport\assetbundles\helpdesksupport\HelpdeskSupportAsset located in ./vendor/jrrdnx/helpdesk-support/src/assetbundles/HelpdeskSupport/HelpdeskSupportAsset.php does not comply with psr-4 autoloading standard. Skipping.`
Because of this bug, the Settings page of plugin is now broken
`Invalid Configuration – yii\base\InvalidConfigException
Failed to instantiate component or class "jrrdnx\helpdesksupport\assetbundles\helpdesksupport\HelpdeskSupportAsset".
↵
Caused by: ReflectionException
Class jrrdnx\helpdesksupport\assetbundles\helpdesksupport\HelpdeskSupportAsset does not exist
in /var/www/craft/vendor/yiisoft/yii2/di/Container.php at line 453`

**Comment by jrrdnx** [comment]
@hoangngt  I renamed that directory to /HelpdeskSupport (from /helpdesksupport) to be compliant back in [1.1.1](https://github.com/jrrdnx/craft-helpdesk-support/releases/tag/1.1.1) so depending on your filesystem it may not have picked up the change. I'd recommend either uninstalling and reinstalling via composer, or remove your /vendors directory and re-running `composer install`.

**Comment by ghost** [comment]
@jrrdnx hmm, it isn't relevant, you forgot changing the path in this twig template
```
10. in /var/www/craft/vendor/jrrdnx/helpdesk-support/src/templates/settings.twig at line 18– craft\helpers\Template::attribute(craft\web\twig\Environment, Twig\Source, craft\web\View, 'registerAssetBundle', ...)
12131415161718192021222324 * @since     0.1.0
 */
#}
 
{% import "_includes/forms" as forms %}
 
{% do view.registerAssetBundle("jrrdnx\\helpdesksupport\\assetbundles\\helpdesksupport\\HelpdeskSupportAsset") %}
 
{{ forms.autosuggestField({
    label: 'provider'|t('helpdesk-support'),
    instructions: 'provider-instructions'|t('helpdesk-support'),
    id: 'apiProvider',
    name: 'apiProvider',
```

**Comment by jrrdnx** [comment]
Interesting. Not able to reproduce locally, but I made sure to update all of those assetbundles paths. Try [1.2.3](https://github.com/jrrdnx/craft-helpdesk-support/releases/tag/1.2.3) and let me know if you have any other issues.

**Comment by ghost** [comment]
@jrrdnx Hi, I tried to update to 1.2.3 but my composer can't recognize this version. Do you have any idea?
` Root composer.json requires jrrdnx/helpdesk-support 1.2.3, found jrrdnx/helpdesk-support[dev-master, dev-develop, 1.0.0, ..., 1.2.2] but it does not match the constraint.
`

**Comment by jrrdnx** [comment]
@hoangngt Should be resolved with [1.2.4](https://github.com/jrrdnx/craft-helpdesk-support/releases/tag/1.2.4)
